occurrences/tests/test_api.py

```diff
@@ -2794,3 +2798,124 @@ def test_unenrol_event_queue_mutation_unauthorized(
     }
     executed = api_client.execute(UNENROL_EVENT_QUEUE_MUTATION, variables=variables)
     assert_permission_denied(executed)
+    assert EventQueueEnrolment.objects.count() == 1
+
+
+def test_pick_enrolment_from_queue(
+    snapshot, staff_api_client, organisation, mock_get_event_data
+):
+    p_event = PalvelutarjotinEventFactory(organisation=organisation)
+    occurrence = OccurrenceFactory(p_event=p_event)
+    queue = EventQueueEnrolmentFactory(p_event=p_event)
+    variables = {
+        "input": {
+            "occurrenceId": to_global_id("OccurrenceNode", occurrence.id),
+            "eventQueueEnrolmentId": to_global_id("EventQueueEnrolmentNode", queue.id),
+        }
+    }
+    staff_api_client.user.person.organisations.add(organisation)
+    executed = staff_api_client.execute(
+        PICK_ENROLMENT_FROM_QUEUE_MUTATION, variables=variables
+    )
+    snapshot.assert_match(executed)
```

Comment:
In the other added tests below you are checking the `Enrolment.objects.count()`, how about checking it here also? Or is there a reason not to check it here? It seems that the count checking is seldom done with snapshot testing in this codebase but it is done e.g. in:
- test_create_event
- test_create_event_with_external_enrolment
- test_create_event_without_enrolment
- test_create_event_without_p_event_translations
- test_delete_event
- test_delete_unpublished_occurrence

What do you think?

Reply:
The snapshot has the count inside it already. I don't have any other explanation for that. It would be good to check the db count because the item can be returned without saving it. That is what happened to me in the last PR. I'll add it in this one at least.